src/components/Icon.tsx

```diff
@@ -21,6 +21,127 @@ function CryptoIcon() {
 }
 Icon.Crypto = CryptoIcon;
 
+function Edit2({
```

Comment:
Do we have a plus icon? This should be that.

Reply:
Can plus icon be used for the edit drop?